test/program_test.cpp

```diff
@@ -160,9 +161,7 @@ TEST_CASE(program_copy)
         auto para1 = mm1->add_parameter("m1", s1);
         auto para2 = mm1->add_parameter("m2", s2);
         auto para3 = mm1->add_parameter("m3", s3);
-        mm1->add_instruction(
-            migraphx::make_op("dot", {{"alpha", 0.31f}, {"beta", 0.28f}}), para1, para2, para3);
-
+        migraphx::add_apply_alpha_beta<float>(*mm1, {para1, para2, para3}, "dot", 0.31f, 0.28f);
```

Comment:
Can the `<float>` be removed? It should deduce the template type.

Reply:
Done